src/frontend/App.js

```diff
@@ -9,8 +9,7 @@ function App() {
     <div className="App">
       <Header />
       <body>
-        <Search search='language'/>
-        <Search search='location'/>
+        <Search />
```

Comment:
I see you changed the way your `Search` component is getting returned.
In order not to have conflicts at the merge stage I would suggest to let @kamilalew to merge her PR first and . you can do it after her.